api/modules/search.py

```diff
@@ -0,0 +1,32 @@
+from flask import request
```

Comment:
It'd be cleaner to organize these imports in a PEP8 style. See: https://www.python.org/dev/peps/pep-0008/#imports